### DISCUSSION TITLE
Horizontal Grid

### DISCUSSION THREAD

**Discussion by TartanLeGrand** [tailwindlabs/tailwindcss#3565] [Help] [answered]
Hello, 

I want to make a horizontal grid identical to the one on https://dribbble.com/shots/4901594-Horizontal-Vertical-Scroll .

I would like it to correspond to the spacing of my 2nd grid which will not be horizontal.

**Comment by alexanderbluhm** [comment, marked as answer]
@TartanLeGrand Does it need to be a grid? Otherwise you could use something like this: https://play.tailwindcss.com/m5Mdw4GN26
